testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -2161,6 +2161,29 @@ testModule('close on target scroll', moduleConfig, () => {
 
 
 testModule('container', moduleConfig, () => {
+    test('wrapper width and height should be restored after container option value changed to window (T937118)', function(assert) {
+        const $container = $('#customTargetContainer');
+        $container.css({
+            width: 100,
+            height: 100
+        });
+
+        const overlay = $('#overlay').dxOverlay({
+            container: $container,
+            visible: true
+        }).dxOverlay('instance');
+
+        let wrapperElement = overlay.$content().parent().get(0);
+
+        assert.strictEqual(wrapperElement.style.width, '100px', 'width is correct');
+        assert.strictEqual(wrapperElement.style.height, '100px', 'height is correct');
+
+        overlay.option('container', null);
+        wrapperElement = overlay.$content().parent().get(0);
```

Comment:
Why do we need to acquire a wrapper element once again?

Reply:
I separated this test to 2 different, so there is no necessity more